cmd/oras/repository/show-tags.go

```diff
@@ -52,6 +55,7 @@ Example - Show tags of the target repository that include values lexically after
 		},
 	}
 	cmd.Flags().StringVar(&opts.last, "last", "", "start after the tag specified by `last`")
+	cmd.Flags().BoolVar(&opts.excludeDigestTag, "exclude-digest-tag", false, "exclude all digest tags created by OCI artifact tag schema")
```

Comment:
nit: maybe "referrers tag schema"?🤔

Reply:
I might need to rephrase this and add example latter, maybe without mentioning `referrer` or `tag schema`, below is why:

The use case of this flag is: one user found that `oras repo tags` is listing out some unexpected tags, he/she want to hide them. The struggling point is: **how does this user know about the unwanted tags**?
- If he/she knows well of the distribution spec, then we only need to make the flag name explainative on the behavior of filtering or hidden
- If he/she knows nothing of the distribution spec(most users don't), then mentioning `referrers` or `tag schema` in the long description won't help them realize that this flag is what they need to use.

Reply:
I agree. The users don't need to know anything about the referrers implementation.

Reply:
Brainstorming: How about "metadata tags"?